app/src/key-manager.ts

```diff
@@ -68,13 +95,13 @@ class KeyManager {
   }
 
   async deletePassword(keyName: string) {
+    const keys = await this._getKeyHash();
     try {
-      const keys = await this._getKeyHash();
       delete keys[keyName];
-      await this._writeKeyHash(keys);
     } catch (err) {
-      this._reportFatalError(err);
```

Comment:
I'm not sure offhand if the try / catch still needs to be here if it's not wrapping the get + write. I think the delete keys[name] will always succeed?

Reply:
I am not sure how JS behaves when trying to delete a non existent key from an object. I added/kept this to prevent issues whith the Mailspring account while working on the migration. I may take another look when fixing the snap issues and see if I can remove it.